2025-05-16 01:26:43,855 INFO    MainThread:3783340 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-05-16 01:26:43,855 INFO    MainThread:3783340 [wandb_setup.py:_flush():67] Configure stats pid to 3783340
2025-05-16 01:26:43,855 INFO    MainThread:3783340 [wandb_setup.py:_flush():67] Loading settings from /sdf/home/c/carsmith/.config/wandb/settings
2025-05-16 01:26:43,855 INFO    MainThread:3783340 [wandb_setup.py:_flush():67] Loading settings from /sdf/home/c/carsmith/PILArNet-ML/3d_segment/wandb/settings
2025-05-16 01:26:43,855 INFO    MainThread:3783340 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-05-16 01:26:43,855 INFO    MainThread:3783340 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /sdf/home/c/carsmith/PILArNet-ML/3d_segment/wandb/run-20250516_012643-5vwa84g6/logs/debug.log
2025-05-16 01:26:43,856 INFO    MainThread:3783340 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /sdf/home/c/carsmith/PILArNet-ML/3d_segment/wandb/run-20250516_012643-5vwa84g6/logs/debug-internal.log
2025-05-16 01:26:43,856 INFO    MainThread:3783340 [wandb_init.py:monkeypatch_ipython():581] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f0098294280>
2025-05-16 01:26:43,856 INFO    MainThread:3783340 [wandb_init.py:init():781] calling init triggers
2025-05-16 01:26:43,856 INFO    MainThread:3783340 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'epochs': 1, 'batch_size': 128, 'lr': 0.0001, '_wandb': {}}
2025-05-16 01:26:43,856 INFO    MainThread:3783340 [wandb_init.py:init():809] starting backend
2025-05-16 01:26:43,856 INFO    MainThread:3783340 [wandb_init.py:init():813] sending inform_init request
2025-05-16 01:26:43,861 INFO    MainThread:3783340 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-16 01:26:43,861 INFO    MainThread:3783340 [wandb_init.py:init():823] backend started and connected
2025-05-16 01:26:43,865 INFO    MainThread:3783340 [wandb_run.py:_label_probe_notebook():1267] probe notebook
2025-05-16 01:26:43,865 INFO    MainThread:3783340 [wandb_run.py:_label_probe_notebook():1277] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-05-16 01:26:43,865 INFO    MainThread:3783340 [wandb_init.py:init():915] updated telemetry
2025-05-16 01:26:43,891 INFO    MainThread:3783340 [wandb_init.py:init():939] communicating run to backend with 90.0 second timeout
2025-05-16 01:26:44,370 INFO    MainThread:3783340 [wandb_init.py:init():1014] starting run threads in backend
2025-05-16 01:26:44,833 INFO    MainThread:3783340 [wandb_run.py:_console_start():2454] atexit reg
2025-05-16 01:26:44,834 INFO    MainThread:3783340 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-05-16 01:26:44,834 INFO    MainThread:3783340 [wandb_run.py:_redirect():2371] Wrapping output streams.
2025-05-16 01:26:44,834 INFO    MainThread:3783340 [wandb_run.py:_redirect():2394] Redirects installed.
2025-05-16 01:26:44,840 INFO    MainThread:3783340 [wandb_init.py:init():1056] run started, returning control to user process
2025-05-16 01:26:44,882 INFO    MainThread:3783340 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-16 01:26:44,882 INFO    MainThread:3783340 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-16 01:27:44,186 INFO    MainThread:3783340 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-16 01:27:44,206 INFO    MainThread:3783340 [wandb_run.py:_finish():2189] finishing run carsmith-stanford-university/simclr_full_unet_training/5vwa84g6
2025-05-16 01:27:44,206 INFO    MainThread:3783340 [jupyter.py:save_history():460] not saving jupyter history
2025-05-16 01:27:44,206 INFO    MainThread:3783340 [jupyter.py:save_ipynb():384] not saving jupyter notebook
2025-05-16 01:27:44,206 INFO    MainThread:3783340 [wandb_init.py:_jupyter_teardown():563] cleaning up jupyter logic
2025-05-16 01:27:44,206 INFO    MainThread:3783340 [wandb_run.py:_atexit_cleanup():2419] got exitcode: 0
2025-05-16 01:27:44,206 INFO    MainThread:3783340 [wandb_run.py:_restore():2401] restore
2025-05-16 01:27:44,206 INFO    MainThread:3783340 [wandb_run.py:_restore():2407] restore done
2025-05-16 01:27:45,048 INFO    MainThread:3783340 [wandb_run.py:_footer_history_summary_info():4064] rendering history
2025-05-16 01:27:45,049 INFO    MainThread:3783340 [wandb_run.py:_footer_history_summary_info():4096] rendering summary
2025-05-16 01:27:45,049 INFO    MainThread:3783340 [wandb_run.py:_footer_sync_info():4025] logging synced files
